source/diskio.c

```diff
@@ -10,8 +10,12 @@
 #include <bdm.h>
 #include <cdvdman.h>
 
+#include <bdm.h>
+#include <cdvdman.h>
+
 #include "ff.h"			/* Obtains integer types */
 #include "diskio.h"		/* Declarations of disk functions */
+

```

Comment:
let's try to avoid dummy changes about empty lines or removing lines as it can create more conflicts with futures versions upgrades

Reply:
I removed all the dummy empty lines.